tests/test_run_reference_vmm.py

```diff
@@ -1,28 +1,22 @@
 # Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
 # SPDX-License-Identifier: Apache-2.0 OR BSD-3-Clause
 """Run the reference VMM and shut it down through a command on the serial."""
+from tools.s3 import download_resource
```

Comment:
nit: Style wise - it's generally preferred to have imports from standard library first, empty line, then third party external  imports, empty line then imports from within the same package/sub-package - so this should go after `import pytest`